04/script.js

```diff
@@ -0,0 +1,323 @@
+const tasksProm = prompt(`Number
+String
+Boolean
+Comparison
+Ternary
+Prompt
+Confirm (or)
+Confirm (if)
+Default (or)
+Default (if)
+Login
+Currency
+Scissors
+чорний пояс
+`)
+
+if (tasksProm === "Number") {
+     // Number: odd
+     const promptValue = prompt('Введіть число');
+
+     if (isNaN(promptValue)) {
+          alert('Ви ввели не число!')
+     }
```

Comment:
I suggest wrapping all block in this if or use else
`if (isNaN(number)) {
    alert("Not a number");
} else {
    if (number % 2 === 0) {
        alert("even");
    } else {
        alert("odd");
    }
}`